charts/telepresence/templates/agentInjectorWebhook.yaml

```diff
@@ -14,7 +15,11 @@ webhooks:
   {{- toYaml . | nindent 2 }}
 {{- end }}
   clientConfig:
-    caBundle: {{ $ca.Cert | b64enc }}
+{{- if and ($secretData) (eq .Values.agentInjector.certificate.regenerate false) }}
```

Comment:
just out of curiosity (not something that you need to change) is there any reason not to use `not .Values.agentInjector.certificate.regenerate` instead of `eq .Values.agentInjector.certificate.regenerate false`?